### PR TITLE
add rule for ComputerWoche.de

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you also search for more subdomains?
We have a guide about how to find them here:
https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md#ruleset-style-guide

**Comment by J0WI** [comment]
@numismatika are you interested in continuing here?

**Comment by J0WI** [comment]
This does not pass our tests.

**Comment by numismatika** [comment]
@J0WI  check again

File: src/chrome/content/rules/ComputerWoche.de.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		- preisvergleich.computerwoche.de
+		- testevent.computerwoche.de
+		- video.computerwoche.de
+		- w3.computerwoche.de
+		- wiki.computerwoche.de
+		- ww.computerwoche.de
+		- www1.computerwoche.de
+		- www2.computerwoche.de
+		- www3.computerwoche.de
+		- wwwl.computerwoche.de
```
Comment (J0WI) [inline src/chrome/content/rules/ComputerWoche.de.xml]: @numismatika could you add a hint about the wildcard DNS record and maybe you can also remove some of the `www`aliases from the list.

**Comment by jeremyn** [comment]
@numismatika Are you still interested in working on this?

**Comment by jeremyn** [comment]
Delete entirely:
- [ ] https://selfcheck.computerwoche.de (connection refused with both HTTPS and plain HTTP)
- [ ] https://servicedesk.computerwoche.de (connection refused with both HTTPS and plain HTTP)
  
Mark as `Invalid certificate`:
- [ ] https://blog.computerwoche.de
- [ ] https://ibmexperts.computerwoche.de
- [ ] https://jobs.computerwoche.de
- [ ] https://juk.computerwoche.de
- [ ] https://w3.computerwoche.de
 
Also:
- [ ] Add a `test` for http://info.datacenter.computerwoche.de/robots.txt
- [ ] Sort everything correctly (see [here](https://github.com/EFForg/https-everywhere/blob/45cf7c85151609862175b09c8ee682d204846465/CONTRIBUTING.md#target-ordering))

**Comment by numismatika** [comment]
Yea the sorting is a bit off when you use sublist3r, i think it sometimes gets confused by multithreading.

**Comment by jeremyn** [comment]
I've updated the checklist through https://github.com/EFForg/https-everywhere/pull/6317/commits/bbd530947c02556db92ffc986f0862fefcd768dd. The sorting is still slightly wrong, specifically `beta.computerwoche.de` should come after `bestinbigdata-de.computerwoche.de`. I also added a new item to change the `ruleset` `name`, and I'm sorry I didn't include that when I first made the checklist.

The sorting in Sublist3r was specifically modified for HTTPS Everywhere, see https://github.com/aboul3la/Sublist3r/pull/38 (by me), later refactored in https://github.com/aboul3la/Sublist3r/pull/56 by @pgerber . What sorting error do you see, specifically?

**Comment by numismatika** [comment]
@jeremyn  the exact sorting problem you see, i imported Sublisters output 1:1 and removed domains that don't have Port 443 open. Every now and then an entry or two are out of order

**Comment by jeremyn** [comment]
Your original PR was from August 2016, but we updated the sorting in Sublist3r in October 2016. You shouldn't see any sorting problems in Sublist3r now. If you do, please let us know or report it at Sublist3r.

**Comment by numismatika** [comment]
i assume this is a side effect of https://github.com/aboul3la/Sublist3r/issues/121

**Comment by jeremyn** [comment]
I've updated the checklist through https://github.com/EFForg/https-everywhere/pull/6317/commits/162ed3c70a4980c5d1082741c3ebe57fb2b6c5aa.

**Comment by jeremyn** [comment]
Thanks, merged.

**Pull request merged**